Configurations/common.tmpl

```diff
@@ -113,7 +123,7 @@
  sub dolib {
      my $lib = shift;
      return "" if $cache{$lib};
-     unless ($disabled{shared}) {
+     unless ($disabled{shared} || $lib =~ /\.a$/) {
```

Comment:
It's just an internal naming convention, and we already use `.o` as a similar convention.  The build files (Makefile et al) then know to convert them to their local convention.

I modeled the internal convention early on the normal Unix extensions because it's shorter to type and, well, I wasn't very imaginative.  I could have chosen `.object` and `.archive` just as well...